src/main/java/org/everit/jira/timetracker/plugin/DurationFormatter.java

```diff
@@ -53,18 +68,29 @@ public DurationFormatter() {
     workHoursPerDay = timeTrackingConfiguration.getHoursPerDay().doubleValue();
   }
 
+  private boolean appendValue(final StringBuilder rval, final Long value, final String key,
+      final boolean nonzeroFragmentVisited) {
+    boolean append = nonzeroFragmentVisited;
+    if (value == null) {
+      return nonzeroFragmentVisited;
+    }
+    if (!nonzeroFragmentVisited && value.longValue() > 0) {
```

Comment:
Is boolean append necessary? nozeroFragmentVisited ||  value.longValue() > 0 would be more simple. Please check.